core/src/test/scala/org/apache/spark/serializer/KryoSerializerSuite.scala

```diff
@@ -62,6 +62,10 @@ class KryoSerializerSuite extends FunSuite with SharedSparkContext {
     val thrown3 = intercept[IllegalArgumentException](new KryoSerializer(conf4).newInstance())
     assert(thrown3.getMessage.contains(kryoBufferProperty))
     assert(!thrown3.getMessage.contains(kryoBufferMaxProperty))
+    val conf5 = conf.clone()
+    conf5.set(kryoBufferProperty, "8m")
```

Comment:
Line 52 does, but I know that's an exception case, though it won't fail as expected unless `m` is handled correctly. OK, it doesn't hurt to add this I suppose.